support/index.adoc

```diff
@@ -33,7 +33,7 @@ endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 ** Roll out {product-title} upgrades.
 ** Improve the upgrade experience.
 
-* *Insight Operator*: By default, {product-title} installs and enables the Insight Operator, which reports configuration and component failure status every two hours. The Insight Operator helps to:
+* *Insights Operator*: By default, {product-title} installs and enables the Insights Operator, which reports configuration and component failure status every two hours. The Insights Operator helps to:
```

Comment:
I am not aware of the context, but it might be possible to turn this (and also same for `Telemetry` on line 30) into one unordered-items list:
```suggestion
* *Insights Operator*: By default, {product-title} installs and enables the Insights Operator, which has the following uses:
** Reports the configuration and component failure status every two hours.
** Identifies potential cluster issues proactively.
** Provides a solution and preventive action in {cluster-manager-first}.
```